CIScripts/Tests/VRouterAgentTests.ps1

```diff
@@ -102,7 +102,7 @@ function Test-VRouterAgentIntegration {
                 return
             }
 
-            Start-Sleep -s $SleepTimeBetweenChecks
+            Start-Sleep -s $TimeBetweenChecksInSeconds
```

Comment:
Why not use `TIME_BETWEEN_AGENT_PROCESS_CHECKS_IN_SECONDS` here and in the other function?